INSTALL.md

```diff
@@ -501,11 +501,21 @@ cd $HOME/Downloads
 wget -q https://www.openssl.org/source/openssl-1.0.2n.tar.gz
 tar xf openssl-1.0.2n.tar.gz
 cd $HOME/Downloads/openssl-1.0.2n
-./Configure --prefix=/usr/local --openssldir=/usr/local linux-x86_64
+./config --shared
```

Comment:
Is /usr/local/ssl the default prefix?  Should we be explicit anyway?

Reply:
If you specify nothing, both `prefix` and `openssldir` will be set to `/usr/local/ssl`. If you specify only `prefix`, it becomes `/usr/local/` and `/usr/local/ssl` respectively. If you specify only `openssldir`, it becomes `/usr/local/` and `/usr/local` respectively. If you specify both, same behavior. Personally I find this behavior to be kind of unintuitive.

Reply:
Would not `./config --openssldir=/usr/local/ssl --shared` do the right thing?

Reply:
I am just going to describe what happens and why the default is a good location in this case. Seems like explicitly setting the destination is involved and more than I feel comfortable describing.

Did I mention that I cannot wait for Ubuntu:14.04 to be EOL?